wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -342,6 +344,7 @@ impl PhysicalDeviceFeatures {
             Df::FRAGMENT_WRITABLE_STORAGE,
             self.core.fragment_stores_and_atomics != 0,
         );
+        dl_flags.set(Df::MULTISAMPLED_SHADING, self.core.sample_rate_shading != 0);
```

Comment:
The other backends should also set this appropriately

Reply:
But I didn't find sample_rate_shading logic at other backends

Reply:
If there is no flag determining if it is supporting, they should always enable it.

Reply:
Checked other backends, all flags are enabled by default like Vulkan backends：
https://github.com/gfx-rs/wgpu/blob/b19000367c15ebc3f4aa51871e6d6b9b670126f8/wgpu-hal/src/metal/adapter.rs#L1036

Reply:
Ahhh I didn't catch that. Cool